input/templates/partials/catalog/suggestions.hbs

```diff
@@ -1,9 +1,4 @@
 <div class="row">
-  <div class="col-sm-12">
-    <p class="text-center text-uppercase may-like">{{i18n "content.suggestions.youMayLike"}}</p>
-  </div>
-</div>
-<div class="row">
 	{{#each content.suggestions.list}}
```

Comment:
I guess the most precise term would be `recommendations`, I don't know, if it's the most generic :)